packages/integration-karma/test/upgradable-element/index.spec.js

```diff
@@ -53,56 +53,3 @@ it('should create elements with correct component behavior even when they share
         'Duplicate Child Component'
     );
 });
-
-// In compat mode, the custom registry is bypassed and an equivalent implementation is used,
-// the native registry restrictions are not applicable
-const SUPPORTS_CUSTOM_ELEMENTS = !process.env.COMPAT && 'customElements' in window;
-if (SUPPORTS_CUSTOM_ELEMENTS) {
-    describe('duplicate registration', () => {
-        function testDuplicateNativeRegistration(testCase, tag, wcClazz) {
-            it(`should throw when registering a duplicate tag in custom element registry with ${testCase}`, () => {
-                // First time the x-interop tag will be registered
-                const interop = createElement(tag, { is: Interop });
-                document.body.appendChild(interop);
-                expect(() => {
-                    customElements.define(tag, wcClazz);
-                }).toThrowError(
-                    /'x-([a-z]|-)*' has already been defined as a custom element|the name "x-([a-z]|-)*" has already been used with this registry|Cannot define multiple custom elements with the same tag name/
-                );
-            });
-        }
-        testDuplicateNativeRegistration(
-            'anonymous class',
-            'x-interop-wc',
-            class extends HTMLElement {}
-        );
-        testDuplicateNativeRegistration(
-            'CustomElementConstructor getter',
-            'x-interop-lwc-constructor',
-            Interop.CustomElementConstructor
-        );
-
-        function testDuplicateLwcRegistration(testCase, tag, wcClazz) {
-            it(`should log error when registering duplicate tag via lwc.createElement to override element registered with ${testCase}`, () => {
-                customElements.define(tag, wcClazz);
-                expect(() => {
-                    const element = createElement(tag, { is: Interop });
-                    // The element will still be created
-                    expect(element.tagName.toLowerCase()).toBe(tag);
-                }).toLogErrorDev(
-                    `Unexpected tag name "${tag}". This name is a registered custom element, preventing LWC to upgrade the element.`
-                );
-            });
-        }
-        testDuplicateLwcRegistration(
-            'anonymous class',
-            'x-interop-native-wc',
-            class extends HTMLElement {}
-        );
-        testDuplicateLwcRegistration(
-            'CustomElementConstructor getter',
-            'x-interop-lwc-wcclass',
-            Interop.CustomElementConstructor
-        );
-    });
-}
```

Comment:
These tests are for registering duplicate tags, which is exactly the thing that pivots allow us to do.